proto/spacemesh/v1/var_types.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
```

Comment:
per our conversation last week, I don't think any of this belongs in the L1 API. I think it belongs at the application layer, and that this data should be returned from this API as a blob of "SVM compatible data"